apisix/plugins/prometheus.lua

```diff
@@ -39,6 +39,7 @@ local _M = {
     name = plugin_name,
     log  = exporter.log,
     schema = schema,
+    run_policy = "prefer_route",
```

Comment:
Does the `nginx-lua-prometheus` need update ?

Reply:
This fix like https://github.com/apache/apisix/pull/4589/files

Reply:
> Does the `nginx-lua-prometheus` need update ?

No, APISIX from 2.7 to master, `nginx-lua-prometheus` has only one version upgrade, and this upgrade of `nginx-lua-prometheus` is not related to the current issue.

https://github.com/knyar/nginx-lua-prometheus/blob/master/CHANGELOG.md#020220127